src/ShellClients/PanelClone.vala

```diff
@@ -136,39 +92,24 @@ public class Gala.PanelClone : Object {
             return;
         }
 
-        clone.visible = true;
+        new GesturePropertyTransition (actor, gesture_tracker, "translation-y", null, calculate_y (true)).start (with_gesture);
 
-        clone.save_easing_state ();
-        clone.set_easing_mode (Clutter.AnimationMode.EASE_OUT_QUAD);
-        clone.set_easing_duration (get_animation_duration ());
-        clone.y = calculate_clone_y (true);
-        clone.restore_easing_state ();
+        gesture_tracker.add_success_callback (with_gesture, () => panel_hidden = true);
     }
 
-    private void show () {
-        if (!panel_hidden) {
+    private void show (GestureTracker gesture_tracker, bool with_gesture) {
+        if (!panel_hidden || last_gesture_tracker.recognizing) {
```

Comment:
same here, PanelClone already has default_gesture_tracker, why do we pass it here again?